### PR TITLE
signature widget

### PR DESCRIPTION
**Pull request by UnsungHero97**
Screen Shot 2020-01-15 at 4 13 39 PM
Screen Shot 2020-01-15 at 4 13 46 PM

### CODE REVIEW

File: src/widgets/signature/SignatureWidget.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+  //       rawDataFormat: 'ENC',
+  //     };
+  //     this.topazElement = document.createElement('TopazElement');
+  //     this.topazElement.setAttribute('messageAttribute', JSON.stringify(message));
+  //     body.appendChild(this.topazElement);
+  //     document.addEventListener('SignResponse', this.handleTopazResponse, false);
+  //     const topazEvent :Event = new Event('SignStartEvent', { bubbles: true, cancelable: false });
+  //     this.topazElement.dispatchEvent(topazEvent);
+  //   }
+  // }
```
Comment (solomontang) [inline src/widgets/signature/SignatureWidget.js]: Don't need to keep this Topaz business around anymore.

File: src/widgets/stories/constants/signatureSchemas.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
+/* eslint-disable max-len */
+
+const dataSchema = {
+  properties: {
+    page1Section1: {
+      properties: {
+        [`${0}`]: {
```
Comment (solomontang) [inline src/widgets/stories/constants/signatureSchemas.js]: Template strings for numbers? '0' should suffice, no?

File: src/fields/ParagraphField.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+const ParagraphTitle = styled.h2`
+  font-size: 16px;
+  font-weight: normal;
+  margin: 16px 0 0 0;
+`;
+
+const ParagraphBody = styled.div`
+  font-size: 14px;
+  margin: 12px 0 0 0;
+`;
```
Comment (solomontang) [inline src/fields/ParagraphField.js]: This should probably be up to the designers and is entirely cosmetic.

What about keeping the margins all in the `ParagraphTitle` and removing the margins in the body? I'm not expecting the margins to collapse, which means the effective distance between the Title and the body is 28px. Seems pretty large?
const ParagraphTitle = styled.div`
  ...
  margin: 16px 0;
`;

Comment (UnsungHero97) [reply src/fields/ParagraphField.js]: Yes, it'd be nice to get input from our designers. However, I don't think designers should be concerned with implementation details, and I would argue that paddings and margins are considered implementation details.

I can go into details about why I chose these specific margins, but here's a visual of the difference between what I have and what you're suggesting:

As is:
Screen Shot 2020-01-28 at 4 03 56 PM

Only `ParagraphTitle` has margins:
Screen Shot 2020-01-28 at 4 04 56 PM

**Pull request merged**